packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md

```diff
@@ -3,7 +3,9 @@
    To handle empty `companyId` values, you can define the ACS URL as:
 
    ```bash
-   ${empty org.companyId ? 'https://login.myapp.io' : 'https://login.myapp.io?connection=' + org.companyId}
+   ${empty org.companyId ? 'login.myapp.io' : 'login.myapp.io?connection=' += org.companyId}
```

Comment:
https:// is missed, the expression should be 
`https://${empty org.companyId ? 'login.myapp.io' : 'login.myapp.io?connection=' += org.companyId}`

Reply:
@vanngo-okta

Reply:
Thanks, I'll correct this in another commit.